src/org/omegat/filters2/moodlephp/MoodlePHPFilter.java

```diff
@@ -167,10 +167,10 @@ protected void processBlock(String block, BufferedWriter out) throws IOException
     }
 
     @Override
-    protected void alignFile(BufferedReader sourceFile, BufferedReader translatedFile, FilterContext fc)
-            throws Exception {
-        Map<String, String> source = new HashMap<String, String>();
-        Map<String, String> translated = new HashMap<String, String>();
+    protected void alignFile(BufferedReader sourceFile, BufferedReader translatedFile, FilterContext fc,
+                             String sourcePath) throws Exception {
```

Comment:
Maybe sourceFile or sourceFilePath instead of sourcePath, to avoid confusion with EntryKey's "path" parameter (whose meaning is not clear, I admit)